kafka-client/src/main/java/dev/responsive/kafka/api/ResponsiveKafkaStreams.java

```diff
@@ -526,29 +522,28 @@ public Params build() {
           streamsConfig,
           storeRegistry,
           metrics,
-          compatibilityMode
+          storageBackend
       );
 
       final var admin = responsiveKafkaClientSupplier.getAdmin(responsiveConfig.originals());
-      if (compatibilityMode == CompatibilityMode.METRICS_ONLY) {
```

Comment:
we should leave this config for now and just have it alias to storage.backend=NONE since some customers and POCs use this